crates/formatter/tests/cases/issue_128/settings.inc

```diff
@@ -0,0 +1,6 @@
+FormatSettings {
+    preserve_breaking_argument_list: true,
+    preserve_breaking_member_access_chain: true,
+    always_break_named_arguments_list: false,
```

Comment:
@innocenzi instantation was always breaking into multiple lines because of this option, not a bug.